entrypoint/mixedoscluster/mixedoscluster_test.go

```diff
@@ -0,0 +1,51 @@
+package mixedoscluster
+
+import (
+	"fmt"
+
+	"github.com/rancher/distros-test-framework/pkg/assert"
+	"github.com/rancher/distros-test-framework/pkg/testcase"
+	"github.com/rancher/distros-test-framework/shared"
+
+	. "github.com/onsi/ginkgo/v2"
+)
+
+var _ = Describe("Test: Mixed OS Cluster", func() {
+
+	It("Starts Up with no issues", func() {
+		testcase.TestBuildCluster(GinkgoT())
+	})
+
+	It("Validates Node", func() {
+		testcase.TestNodeStatus(
+			assert.NodeAssertReadyStatus(),
+			nil,
+		)
+	})
+
+	It("Validate Pods", func() {
+		testcase.TestPodStatus(
+			assert.PodAssertRestart(),
+			assert.PodAssertReady(),
+			assert.PodAssertStatus(),
+		)
+	})
+
+	It("Validates internode connectivity over the vxlan tunnel", func() {
+		testcase.TestInternodeConnectivityMixedOS()
+	})
+
+	It("Validates cluster by running sonobuoy mixed OS plugin", func() {
+		testcase.TestSonobuoyMixedOS(sonobuoyVersion, true)
+		shared.ManageWorkload("delete", "",
```

Comment:
the point is not that is not needed....is that is a parameter that we shouldnt passing around empty since we always have a default value. 

if the point is having this passing as empty , when u pass the flag to start the arch == arm, how does this parameter will know that ? 

even if this will not be working for arm.

some examples 

```
func TestIngress(deployWorkload bool) {
	if deployWorkload {
		_, err := shared.ManageWorkload(
			"create",
			"ingress.yaml",
			customflag.ServiceFlag.ClusterConfig.Arch.String(),
		)
		Expect(err).NotTo(HaveOccurred(),
			"Ingress manifest not deployed")
	}
```

```	
if deployWorkload {
		_, err := shared.ManageWorkload(
			"create",
			"local-path-provisioner.yaml",
			customflag.ServiceFlag.ClusterConfig.Arch.String(),
		)
		Expect(err).NotTo(HaveOccurred(),
			"local-path-provisioner manifest not deployed")
	}
```

```
func TestServiceNodePort(deployWorkload bool) {
	if deployWorkload {
		_, err := shared.ManageWorkload(
			"create",
			"nodeport.yaml",
			customflag.ServiceFlag.ClusterConfig.Arch.String(),
		)
		Expect(err).NotTo(HaveOccurred(),
		
```

Reply:
on the coredns.go file, i am passing it wrongly 
```
	if deployWorkload {
		_, err := shared.ManageWorkload("create", "dnsutils.yaml", "arch")
		Expect(err).NotTo(HaveOccurred(),
			"dnsutils manifest not deployed", err)
	}

```

can u fiz for me in your PR ?

Reply:
Lets discuss this offline or on standup